src/validation/userValidation.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable require-jsdoc */
+import validator from 'validatorjs';
+import userService from '../services/userService';
+
+
+/**
+ * UsersValidation
+ */
+
+class UsersValidation {
+  /**
+   * @returns {object} ValidateUserSignIn
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   */
+  static async ValidateUserSignIn(req, res, next) {
+    let { email, password } = req.body;
+    const constraint = {
+      email: 'required|email|min:12|max:30',
+      password: 'required|min:8|max:14|alpha_num',
+    };
+
+    // eslint-disable-next-line new-cap
+    const validation = new validator(req.body, constraint);
```

Comment:
Class names should start with capital letter.

Reply:
ok, Done